lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -83,6 +83,8 @@ export abstract class ClientApplication {
     // Flag representing whether or not the initialize API has been called and completed
     protected initialized: boolean;
 
+    private ssoSilentMeasurement?: InProgressPerformanceEvent = undefined;
```

Comment:
The '?' takes care of this for you
```suggestion
    private ssoSilentMeasurement?: InProgressPerformanceEvent;
```

Reply:
fixed